### DISCUSSION TITLE
Website made with nuxt.js

### DISCUSSION THREAD

**Discussion by jatukwase** [nuxt/nuxt#24722] [General]
How can i get a script website made with nuxt.js example website https://e-ttiktok[dot]com/

![Screenshot 2023-12-13 132844](https://github.com/nuxt/nuxt/assets/153725742/c36a5b51-a942-4ebf-ab07-941c3d257793)

**Comment by xjccc** [comment]
[nuxt templates](https://nuxt.com/templates)
